### PR TITLE
Raunaq form

### PR DESCRIPTION
**Pull request by Raunaq27**
Finished the basic setup of (#16)  
It generates questions that can be used in the other pages we have made so far

### CODE REVIEW

File: docs/script.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+window.addEventListener("DOMContentLoaded", (event) => {
+  console.log("DOM fully loaded and parsed");
+
+  let idtAddBtn = document.getElementById('idt-add-btn');
+
+  let idtMain = document.getElementById('idt-QAs');
+
+  idtAddBtn.addEventListener("click", (event) => {
+    idtMain.innerHTML += `<br>
```
Comment (FeXd) [inline docs/script.js:9]: Add indentation so this is easier to read.

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <section id="idt-form">
+        <h1>Interactive Learning Generator</h1>
+        <div>
+            <h3>A student project by <a href="https://github.com/Raunaq27" target=”_blank”>@Raunaq27</a> & <a
+                    href="https://github.com/Anmol135" target=”_blank”>@Anmol135</a> </h3>
+                    <a href="https://github.com/Interactive-Design-and-Technology/interactive-learning" target="_blank">Github link</a>
+            <p>This tool allows you to generate HTML, CSS, & JavaScript for interactive learning questions that can be
+                inserted into learning management systems.</p>
+            <form>
+                <label for="type">Type:</label>
```
Comment (FeXd) [inline docs/index.html:20]: Validation error: `The value of the for attribute of the label element must be the ID of a non-hidden form control.`

c/o https://validator.w3.org/

File: docs/script.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    <label for="question">Q:
```
Comment (FeXd) [inline docs/script.js:10]: Validation error: "The value of the for attribute of the label element must be the ID of a non-hidden form control."

File: docs/style.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+#idt-form {
+    background-color: darkgray;
+    max-width: 50%;
```
Comment (FeXd) [inline docs/style.css:3]: Consider adding media query so that 100% width is allowed after a specific break.

File: docs/style.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    margin: 0 25% 0 25%;
+}
+
+
+#idt-form a {
+    color: #000;
+}
+
+footer {
```
Comment (Raunaq27) [inline docs/style.css:31]: There is no actual footer on the page, needs to be removed

**Review by Raunaq27 — COMMENTED** [review]
need to remove the footer from the css

**Comment by FeXd** [comment]
@Raunaq27 Looking at the latest deployment, I think there should also be a way to REMOVE questions?

**Review by FeXd — APPROVED** [review]

**Pull request merged**